api_app/analyzers_manager/file_analyzers/doc_info.py

```diff
@@ -192,6 +207,53 @@ def analyze_for_cve(self) -> List:
                     )
         return results
 
+    def get_external_relationships(self) -> List:
+        external_relationships = []
+        oid = oleid.OleID(self.filepath)
+        if sum([i.value for i in oid.check() if i.id == "ext_rels"]) > 1:
+            xml_parser = XmlParser(self.filepath)
+            for relationship, target in oleobj.find_external_relationships(xml_parser):
+                external_relationships.append(
+                    {
+                        "relationship": relationship,
+                        "target": target,
+                    }
+                )
+        return external_relationships
+
+    def get_docx_urls(self) -> List:
+        urls = []
+
+        try:
+            document = zipfile.ZipFile(self.filepath)  # check if docx document
+            dxml = document.read("docProps/app.xml")
+            pages_count = (
+                minidom.parseString(dxml)
+                .getElementsByTagName("Pages")[0]
+                .childNodes[0]
+                .nodeValue
+            )
+        except Exception:
+            return []
```

Comment:
we can log it as a warning and add it in the "errors" attribute of the report so that it comes visible in the output